docs/tutorials/ml_inference/semantic_search/optimize_vector_search_with_cohere_compressed_embedding.md

```diff
@@ -0,0 +1,536 @@
+  }
+}
+```
+Sample response
+```
+{
+  "task_id": "COP0OZUB3JwAtE25yiQr",
+  "status": "CREATED",
+  "model_id": "t64OPpUBX2k07okSZc2n"
+}
+```
+
+Let's test our model to ensure it's working correctly:
+
+```
+POST _plugins/_ml/models/t64OPpUBX2k07okSZc2n/_predict
+{
+  "parameters": {
+    "texts": ["Say this is a test"],
+    "embedding_types": [
+      "float", "int8"
```

Comment:
The tutorial is for quantized embedding, so should we remove float type to avoid distracting users?

Reply:
make sense